vhost-device-scmi/src/devices/common.rs

```diff
@@ -224,19 +224,18 @@ pub fn devices_help() -> String {
 #[derive(Debug)]
 pub struct Sensor {
     /// The sensor name (possibly truncated) as reported to the guest.
-    pub name: String,
     /// Whether the sensor is enabled.
+    pub name: Option<String>,
```

Comment:
Why we are moving this under the comment of the next option?

Reply:
For each scmi device, it should have name, but there has several ways to get name.

> 1. Use name parameter from user's command.
> 2. Use sysfs or other native attributes.
> 3. Use default name

At the beginning of creating device, if no user's command with name, we don't know whether default name should be set, because we have not scanned the sysfs to find whether there exist name attritbutes yet.
So, after discussing with [mz-pdm](https://github.com/mz-pdm), we think it's better to change this var to Option, so that it can help us to judge whether name has been set when register.

Reply:
I'm just asking why moving that line in the comment on the next field of the structure.
```
@@ -224,8 +224,8 @@ pub fn devices_help() -> String {
 #[derive(Debug)]
 pub struct Sensor {
     /// The sensor name (possibly truncated) as reported to the guest.
-    pub name: String,
     /// Whether the sensor is enabled.
+    pub name: Option<String>,
     ///
     /// Sensors can be enabled and disabled using SCMI.  [Sensor]s created
     /// using [Sensor::new] are disabled initially.
```

Reply:
Oh, sorry. It is a mistake, I will modify it.

Reply:
In branch from [1fb2f52 ](https://github.com/rust-vmm/vhost-device/commit/1fb2f52e500f9fe6821e31a54fdf1f8c231c9b72)to [533416e](https://github.com/rust-vmm/vhost-device/commit/533416e21d45c7cc4b82072e6a146ac9bb82456f), I fix this mistake.